src/main/java/org/opensearch/geospatial/search/aggregations/bucket/geogrid/GeoHexHelper.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.geospatial.search.aggregations.bucket.geogrid;
+
+import static org.opensearch.geospatial.h3.H3.MIN_H3_RES;
+import static org.opensearch.geospatial.h3.H3.h3IsValid;
+import static org.opensearch.geospatial.h3.H3.h3ToLatLng;
+import static org.opensearch.geospatial.h3.H3.stringToH3;
+
+import java.util.Locale;
+
+import lombok.NonNull;
+
+import org.opensearch.common.geo.GeoPoint;
+import org.opensearch.geospatial.h3.H3;
+
+public class GeoHexHelper {
+
+    /**
+     * Checks whether given precision is within H3 Precision range
+     * @param precision H3 index precision
+     */
+    public static void checkPrecisionRange(int precision) {
+        if ((precision < MIN_H3_RES) || (precision > H3.MAX_H3_RES)) {
```

Comment:
Can we do static import for MAX_H3_RES as well?

Reply:
Ack